Mathlib/Data/Set/Intervals/Disjoint.lean

```diff
@@ -253,4 +255,26 @@ theorem iUnion_Iic_eq_Iic_iSup {R : Type*} [CompleteLinearOrder R] {f : ι → R
   @iUnion_Ici_eq_Ici_iInf ι (OrderDual R) _ f has_greatest_elem
 #align Union_Iic_eq_Iic_supr iUnion_Iic_eq_Iic_iSup
 
+theorem iUnion_Iio_of_not_bddAbove_range (hf : ¬ BddAbove (range f)) :
```

Comment:
Please add order-dual versions (probably using the `OrderDual` trick).

Reply:
I thought the version below, now renamed  `iUnion_Iio_eq_univ_iff` and `iInter_Iio_eq_empty_iff` are duals of each other. Do you mean some other sort of dual statement? If yes, which one?

Reply:
ping @urkud